config/locales/en.yml

```diff
@@ -62,6 +62,9 @@ en:
     url: https://ico.org.uk
   tell_employee_about_criminal_record:
     url: https://www.gov.uk/exoffenders-and-employment
+  realistic_job_previews:
+    url_apply_again: https://teacherselect.qualtrics.com/jfe/form/SV_4VHMkFk6TeonVeS
+    url_get_school_experience: https://teacherselect.qualtrics.com/jfe/form/SV_aXbiUM3xz0uVtuS
```

Comment:
It feels like this needs a tidy up. I included this lin in the trello ticket for completeness, but we don't actually use it or need it in the Apply codebase. I'll create a PR to remove.